### ISSUE TITLE
'evidence' - Attribute does not exist

### ISSUE THREAD

**Issue by ntung** [saezlab/pypath#137]
Hi all,

I am trying to fix tests, especially `test_network_graph.py`. The first test `test_complex_expansion` is passed while the test `test_orthology_translation` is failed. The log reveals the lines below.
```
new_edge['dirs'] = e['dirs'].translate(ids)
new_edge['refs_by_dir'] = (self._translate_refsdir(e['refs_by_dir'], ids))
new_edge['evidence'] = e['evidence']
```
The error (caused from the line `new_edge['evidences'] = e['evidences']`) is:
```
KeyError: Attribute does not exist.
```

Any idea to fix that?

Thanks,
Tung

**Comment by Nic-Nic** [comment]
Hi Tung,

I refer to my previous answer regarding `igraph` and tests:https://github.com/saezlab/pypath/issues/132#issuecomment-589590655

**Comment by ntung** [comment]
Hi @Nic-Nic,

IMHO, I don't think it is the good way of communicating when I have just created a ticket, got your replied message while I haven't verified it. However, the ticket is closed straightaway.

By the way, this error caused in the core classes named `legacy.main`. I agree that the tests are far away of the current coverage. However, the core classes should be fine to be used and programmed. Where am I wrong here?

Best,
Tung

**Comment by deeenes** [comment]
Hi Tung,

Thanks for your report, this was indeed an old name of that attribute, now it's called `attrs`.
Fixed in bfdaa7e

Best,

Denes

**Comment by ntung** [comment]
Thanks for your kind reaction! @deeenes 

All the best,
Tung
